From 2d8f1d789a51088dd2f4f2840734f70fb2275cf1 Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Fri, 4 Sep 2020 15:41:29 -0700
Subject: [PATCH] runtime-install: don't install notification-daemon

libnotify dropped the requirement just a couple of months after
this line was added, but we never took it back out again.

Reference:https://github.com/weldr/lorax/commit/a87f438af10e308eb20f2eb2712abe3f820c674a
The patch was adapted to remove the useless deletion of notification-daemon in runtime-cleanup.tmpl

Signed-off-by: Adam Williamson <awilliam@redhat.com>
Signed-off-by: Qiumiao Zhang <zhangqiumiao1@huawei.com>
---
 share/templates.d/99-generic/runtime-cleanup.tmpl | 2 --
 share/templates.d/99-generic/runtime-install.tmpl | 3 ---
 2 files changed, 5 deletions(-)

diff --git a/share/templates.d/99-generic/runtime-cleanup.tmpl b/share/templates.d/99-generic/runtime-cleanup.tmpl
index d803ee0..f3a7ebb 100644
--- a/share/templates.d/99-generic/runtime-cleanup.tmpl
+++ b/share/templates.d/99-generic/runtime-cleanup.tmpl
@@ -44,8 +44,6 @@ removefrom initscripts /usr/sbin/* /usr/share/locale/* /usr/share/doc/* /usr/sha
 removepkg pinentry
 ## no storage device monitoring
 removepkg device-mapper-event dmraid-events sgpio
-## no notifications in anaconda
-removepkg notification-daemon
 ## logrotate isn't useful in anaconda
 removepkg logrotate
 remove /etc/logrotate.d
diff --git a/share/templates.d/99-generic/runtime-install.tmpl b/share/templates.d/99-generic/runtime-install.tmpl
index 729a55a..b6cf765 100644
--- a/share/templates.d/99-generic/runtime-install.tmpl
+++ b/share/templates.d/99-generic/runtime-install.tmpl
@@ -182,8 +182,5 @@ installpkg wget rsync bind-utils ftp mtr vconfig
 installpkg icfg spice-vdagent
 installpkg gdisk hexedit sg3_utils
 
-## satisfy libnotify's desktop-notification-daemon with the least crazy option
-installpkg notification-daemon
-
 ## actually install all the requested packages
 run_pkg_transaction
-- 
2.28.0.windows.1

